tests/python/frontend/onnx/test_forward.py

```diff
@@ -4204,15 +3932,12 @@ def verify_softplus(indata):
     test_pad()
     test_split()
     test_binary_ops()
-    test_single_ops()
+    test_unary_ops()
     test_leaky_relu()
     test_elu()
     test_selu()
     test_prelu()
     test_ThresholdedRelu()
-    test_ScaledTanh()
-    test_ParametricSoftplus()
-    test_Scale()
```

Comment:
Note that I removed these tests because they don't correspond to real onnx operators.